dev/tasks/generate-crds

```diff
@@ -24,7 +24,7 @@ cd ${REPO_ROOT}
 
 # Run controller-gen to generate CRDs
 cd ${REPO_ROOT}/apis
-go run sigs.k8s.io/controller-tools/cmd/controller-gen@v0.14.0 object crd:crdVersions=v1,allowDangerousTypes=true output:crd:artifacts:config=config/crd/ paths="./..."
+go run sigs.k8s.io/controller-tools/cmd/controller-gen@master object crd:crdVersions=v1,allowDangerousTypes=true output:crd:artifacts:config=config/crd/ paths="./..."
```

Comment:
Do we want to keep this pinned ?

Reply:
Yes, but we need a PR.  I sent https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1868 to add a TODO